src/WebJobs.Script/Sanitizer.cs

```diff
@@ -30,6 +30,14 @@ internal static string Sanitize(string input)
                 return string.Empty;
             }
 
+            // Everything we *might* replace contains an equal, so if we don't have that short circuit out.
+            // This can be likely be more efficient with a Regex, but that's best done with a large test suite and this is
+            // a quick/simple win for the high traffic case.
+            if (!input.Contains("="))
```

Comment:
If someone were to add a new pattern to CredentialTokens (e.g. "Secret:") this check would bypass that. To be future safe, we should add a meta test ensuring that all CredentialTokens contain this skip token.

Reply:
Good idea - will do!

Reply:
I went with `MayContainCredentials` (better ideas?) after coding it up here to better reflect the meaning (I think) and test added to ensure we don't regress here.